index.js

```diff
@@ -2,14 +2,65 @@ const Vis = require("./graphvis");
 const { Module, render } = require("./visRenderer");
 
 let Sequelize;
-const relationship = ({ source, target, associationType, as }) => {
-  const typeString = {
-    BelongsTo: `[arrowtail=crow, arrowhead=none, dir=both, arrowsize=0.60]`,
-    BelongsToMany: `[arrowtail=crow, arrowhead=crow, dir=both, arrowsize=0.60]`
-  }[associationType];
-  if (typeString) {
-    return `"${source.name}" -> "${target.name}" ${typeString}`;
-  }
+
+const relationships = (associations, customArrowShapes = {}, arrowSize = 0.6) => {
+  const arrowShapes = Object.assign({
+    BelongsToMany: ['none', 'crow'],
+    BelongsTo: ['crow', 'none'],
+    HasMany: ['none', 'crow'],
+    HasOne: ['none', 'none'],
+  }, customArrowShapes);
+
+  const groupedAssociations = associations.reduce((result, association) => {
```

Comment:
For readability I'd like to use lodash's group by

Reply:
I've used lodash's `groupBy()` in the next commit!